### DISCUSSION TITLE
Support for post upgrade commands in hosted Github app

### DISCUSSION THREAD

**Discussion by devversion** [renovatebot/renovate#11660] [Request Help]
The Renovate documentation specifies the `postUpgradeTasks` option that allows for arbitrary commands to execute before the Renovate upgrade PR is being created. This setting does not seem to work for the Github hosted app from Renovate. 

is it possible that this option will only work for self-hosted instances of Renovate? The documentation does not seem to specify that: https://docs.renovatebot.com/configuration-options/#postupgradetasks. Any help would be appreciated.

**Comment by viceice** [comment]
It's only for self-hosted, you need to configure `allowedPostUpgradeCommands`, which is a global only option.

![image](https://user-images.githubusercontent.com/1798109/132714373-60d367d4-2f72-427e-82ee-2146dcabf2f1.png)

**Reply by devversion** [reply]
I see. thanks for clarifying that!
